subsys/disk/disk_access_spi_sdhc.c

```diff
@@ -768,9 +787,20 @@ static int sdhc_spi_init(struct device *dev)
 
 	data->spi = device_get_binding(DT_BUS_LABEL(DT_INST(0, zephyr_mmc_spi_slot)));
 
-	data->cfg.frequency = SDHC_SPI_INITIAL_SPEED;
-	data->cfg.operation = SPI_WORD_SET(8) | SPI_HOLD_ON_CS;
-	data->cfg.slave = DT_REG_ADDR(DT_INST(0, zephyr_mmc_spi_slot));
+	/* SPI config for initialization (default) */
+	sdhc_select_spi_cfg(data, SDHC_SPI_INIT_CFG);
+	data->spi_cfgs[SDHC_SPI_INIT_CFG] = (struct spi_config){
```

Comment:
I'm OK with this style, but I haven't seen it used much in the rest of Zephyr.  People seem to prefer one line per member when this is inside a function.

No need to do anything unless someone else comments.

Reply:
I think this style reads more naturally here, and is consistent with use like
```
var = (struct k_whatever)K_WHATEVER_INITIALIZER();
```